wordlist.py

```diff
@@ -49,6 +49,51 @@ class MissingTranslationException(Exception):
             'mean',
             'poor',
         ]
+    ],
+	"de_DE": [
+        [
+            'Der Raum',
+            'Bytespeicher'
+        ],
+        [
+            'ist',
+        ],
+        [
+            'offen',
+            'Zutritt möglich',
+        ],
+        [
+            'geschlossen',
+            'nicht mehr geöffnet'
+        ],
+        [
+            'großartig',
```

Comment:
These need to be capitalized as they are the beginning of a new sentence. See https://twitter.com/bytestatus for examples.